win32ss/drivers/font/ftfd/rosglue.c

```diff
@@ -127,3 +137,10 @@ fclose(FILE *Stream)
     DPRINT1("Doubleplus ungood: freetype shouldn't fclose!\n");
     return EOF;
 }
+
+/* Getting environment variable */
+char *getenv(const char *name)
```

Comment:
What build error do you get if you don't include a getenv stub?
(It is used by the ft_getenv macro, which is used only when any of `FT_CONFIG_OPTION_ENVIRONMENT_PROPERTIES` or `FT_DEBUG_MEMORY` or `FT_DEBUG_LEVEL_TRACE` are defined in the ftoption.h ; however they all seem to be disabled...)

Reply:
Deleted successfully.